src/runtime/virtcontainers/nydusd_linux.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2017 Intel Corporation
```

Comment:
This file and `nydusd_other.go` have different copyright headers.

Reply:
The other file was just work moved to a new spot, does that constitute a new header?

Reply:
agreed.  I think what Danny has here makes sense @liubin.  _other is a new file, whereas _linux is just a move of existing code. I would follow same pattern?

Reply:
@dcantah @egernst Ok.